pkgs/by-name/ws/wstunnel-rust/package.nix

```
@@ -0,0 +1,32 @@
+{ fetchFromGitHub
+, rustPlatform
+, lib
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "wstunnel-rust";
+  version = "9.6.1";
+
+  src = fetchFromGitHub {
+    owner = "erebe";
+    repo  = "wstunnel";
+    rev = "v${version}";
+    sha256 = "sha256-+tNQf53KUNIz9RFVTHqgXEKLxlXiQKKPZtXqD3C/t7I=";
+  };
+
+  cargoHash = "sha256-NER7NIWUXyWFUGcdZ5HLFaKaO9FNLPlG7jql08karS4=";
+  checkFlags = [
+    # make use of network connection
+    "--skip=tcp::tests::test_proxy_connection"
+  ];
+
+  meta = with lib; {
+    description = "Wstunnel uses the websocket protocol which is compatible with http in order to bypass firewalls and proxies. Wstunnel allows you to tunnel whatever traffic you want and access whatever resources/site you need.";
```

```suggestion
    description = "Wstunnel uses the websocket protocol in order to bypass firewalls and proxies";
```
we neeed to shorten that or move some things to longDescription.